.github/workflows/imagebuild.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
     strategy:
       matrix:
-        rust_version: ["1.75", "1.81", "1.84", "1.85"]
+        rust_version: ["1.82", "1.85", "1.86"]
```

Comment:
Can this be 1.81? That's the MSRV we have in geo and proj.

Reply:
Ah, I didn't see your comments in https://github.com/georust/docker-images/pull/41. Yeah, let's stick with 1.81 - 1.86 then. This proj situation is such a slog.